src/index.ts

```diff
@@ -63,6 +63,9 @@ type TimeoutResult = {
 }
 
 export const runAll = async (event: AwsEvent): Promise<Output[]> => {
+  if (event.library && event.library.chapter >= 2) {
+    require('./list.js')
```

Comment:
I think the list library should be available for graders with external libraries, even those using chapter 1 (e.g runes and curves). Could the chapter check be removed in that case?

Reply:
Ok. Though, neither runes nor curves use list.js, and list is not in Source chapter 1.